infrastructure_playbook_repo.md

```diff
@@ -0,0 +1,179 @@
+* Galaxy EU compute infrastructure is run on the [BW OpenStack cloud](https://portal.bw-cloud.org/). At the time of writing (21/02/2023) our cloud is of size 8488 VCPUs, 44.6 TB RAM, 162.6 TB storage. Additionally, a few petabytes of [storage](https://github.com/usegalaxy-eu/operations/blob/main/storage.md) is also mounted (NFS) in the cloud.
+* The compute infrastructure (cloud cluster) is configured through [VGCN infrastructure repo](https://github.com/usegalaxy-eu/vgcn-infrastructure) where we define what [cloud images](https://github.com/usegalaxy-eu/vgcn) should be used, the size of the cloud cluster, the number of VMs, the cloud network, the cloud security groups, etc.
+* The cloud is configured through this [infrastructure repo](https://github.com/usegalaxy-eu/infrastructure) using [Terraform](https://www.terraform.io/). The underlying cloud hardware, storage, network, etc are managed by the compute center of the University of Freiburg. For DNS records we use [Amazon's Route53](https://aws.amazon.com/route53/).
+* Some documentation related to [services](https://github.com/usegalaxy-eu/operations/blob/main/cloud/services.md) and IT operations are available in this [operations repo](https://github.com/usegalaxy-eu/operations)
+* For Galaxy Admin training you can refer [here](https://training.galaxyproject.org/training-material/topics/admin/)
+* For monitoring of the Galaxy EU infrastructure we use [Grafana](https://grafana.com/). The dashboards are available [here](https://stats.galaxyproject.eu/)
+
+## Ansible
+* [Ansible](https://www.ansible.com/) is an open-source software provisioning, configuration management, and application-deployment tool enabling infrastructure as code.
+* The basic components of Ansible can be found [here](https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html). Understanding this is important to understand this repo.
+
+## Our repo structure
+* [files](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/files): contains files/configs that are used by the playbooks/roles
+* [group_vars](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/group_vars) and [host_vars](https://github.com/usegalaxy-eu/infrastructure-playbook/tree/master/host_vars): contains variables that are used by the playbooks specific to certain host/group defined in the [inventory file](https://github.com/usegalaxy-eu/infrastructure-playbook/blob/master/hosts). For every playbook we have an associated group_vars/host_vars file where we define the variables that are used by the playbook and the roles that are included/imported in the playbook.
```

Comment:
We don't use host_vars and in my understanding it can be deleted, but we should double check this

Reply:
That's my understanding too, while doing a complete cleaning we can revisit this and fix it.